torch/csrc/jit/mobile/profiler_edge.cpp

```diff
@@ -0,0 +1,48 @@
+#include <torch/csrc/jit/mobile/profiler_edge.h>
+
+namespace profiler = torch::autograd::profiler;
+namespace torch {
+namespace jit {
+namespace mobile {
+
+KinetoEdgeCPUProfiler::KinetoEdgeCPUProfiler(
+    const torch::jit::mobile::Module& m,
+    const std::string& fname,
+    const bool report_input_shapes,
+    const bool profile_memory,
+    const bool with_stack,
+    const bool with_flops,
+    const bool with_module_hierarchy) :
+  m_(m), trace_file_name_(fname) {
```

Comment:
The dependency to m I don't quite understand well.
In the docs you said that m_ exists to make an implicit dependency on the Module explicit, but can you clarify what's the implicit dependency coming from?

Reply:
OK I think these comments went unnoticed due to the code being outdated. So implicit dependency is coming from the fact that post processing callback lambda captures m, so we dont really need to explicitly capture it as part of EdgeProfiler.